raiden/network/transport/matrix/layer.py

```diff
@@ -99,9 +106,81 @@ def full_node(self) -> TransportNode:
     def light_clients(self) -> List[TransportNode]:
         return self._light_clients
 
-    @staticmethod
-    def new_light_client(address: Address, config: dict, auth_params: dict) -> TransportNode:
-        return MatrixLightClientTransportNode(address, config, auth_params)
+    def light_client_onboarding_data(self, address: typing.Address) -> dict:
+        # fetch list of known servers from raiden-network/raiden-tranport repo
+        available_servers_url = DEFAULT_MATRIX_KNOWN_SERVERS[self.environment_type]
+        available_servers = get_matrix_servers(available_servers_url)
+
+        client = make_client(available_servers)
+        server_url = client.api.base_url
+        server_name = urlparse(server_url).netloc
+        return {
+            "display_name_to_sign": "@" + to_normalized_address(address) + ":" + server_name,
+            "password_to_sign": server_name,
+            "seed_retry": "seed",
+        }
+
+    def register_light_client(self, raiden_api: RaidenAPI, registration_data: dict) -> TransportNode:
+        config = raiden_api.raiden.config["transport"]["matrix"]
```

Comment:
This lgtm for now but i think we can modify this method in a near future this way: 

- receive `Wal `and `config `instead of `raiden_api`
- move the `save_light_client `method to light_Client_service
- remove the `start_transport_on_runtime` method and just invoke transport.start

this way we can remove the `RaidenApi `param and dependency.

Reply:
https://jirainfuy.atlassian.net/browse/RLP-858

Reply:
thanks, agreed.